apps/server/src/shared/common/error/invalid-operation.error.ts

```diff
@@ -0,0 +1,16 @@
+import { HttpStatus } from '@nestjs/common';
+import { BusinessError } from './business.error';
+
+export class InvalidOperationError extends BusinessError {
```

Comment:
"Invalid" sounds like a bad request 400 to me.
I'd prefer to call this a
```suggestion
export class ForbiddenOperationError extends BusinessError {
```

Reply:
Agreed. We had taken this from the Microsoft docs. Yet, the original meaning is also referring to an technical invalid state rather then a business error, which indeed is rather forbidden than illegal.